packages/angular/src/2016-12-01/button-directive.js

```diff
@@ -0,0 +1,58 @@
+/**
```

Comment:
I think we should make this a component instead of a directive: https://docs.angularjs.org/guide/component

What do you think?